build.xml

```diff
@@ -2300,7 +1861,7 @@
 
   <!-- Installs artifacts to local Maven repository -->
   <target name="mvn-install"
-          depends="maven-declare-dependencies,jar,sources-jar,javadoc-jar"
+          depends="jar,sources-jar,javadoc-jar"
           description="Installs the artifacts in the Maven Local Repository">
```

Comment:
I'm not sure the references in this target are correct -- does anyone have pointers on how to make sure install and publish are working correctly?

Reply:
References are not correct. Each `pomFile` needs to point to the new corresponding pom file location and name (under `.build/`)

test with `ant mvn-install` and check your local `~/.m2/repository/org/apache/cassandra/`